### ISSUE TITLE
[Bug report]: ENOENT, public/build/index.node not found

### ISSUE THREAD

**Issue by asiyanko** [iotaledger/firefly#2766]
### Platform

Desktop

### Version

1.4.1

### Description

The newest version of the FireFly 1.4.1 produces error box upon start.
I run CentOS Stream 8 Linux under VirtualBox.

### Expected behaviour

Normal start of the application.

### Actual behaviour

FieFly starts and I can see blank main window, Then pop-up box with error opens (see Diagnostics section). When I close the box I can only see blank Firefly window. None of the main menu Items under "Firefly" works except "About FireFly" and "Quit"

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

Just run the application.

### Operating System

Linux

### Diagnostics

```shell
App Version: 1.4.1

platform: linux
platformVersion: 4.18.0-373.el8.x86_64
platformArchitecture: x64
cpuCount: 4
totalMem: 5747.4 MB
freeMem: 1217.8 MB

Error Type: [Preload Context] Error

Error: node-loader:
Error: ENOENT, public/build/index.node not found in /tmp/.mount_fireflJpUgi5/resources/app.asar
```

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

Previously staked

### Crash Status

White screen

### Error Messages

See Diagnostics section.

### Error Logs

```shell
Not available (menu item doesn't work)
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by asiyanko** [comment]
FYI: I use firefly-desktop-1.4.1.appImage to run the app.

**Comment by asiyanko** [comment]
IMPORTANT UPDATE: I was only able to launch FireFly 1.4.1 (as well as 1.4.2) under Linux after I've removed ~/.config/FireFly  folder and reinitialized the application from scratch via a pass phrase.

**Comment by rajivshah3** [comment]
Hmm, that's interesting. I'm not sure why this error happened

**Comment by asiyanko** [comment]
FYI: I've tried to replay the situation. I've renamed ~/.config/FireFly folder, created by FF 1.4.2, to something else, then downloaded FF 1.3.3 AppImage again and reinitialized it using a pass phrase. Then I've launched FF 1.4.2 AppImage over freshly created ~/.config/FireFly. This time though upgrade went smoothly without any errors. Have no idea how that happened previous time, but it is impossible for me to reproduce the error once again and I was not smart enough to save previous version of the ~/.config/FireFly folder. Anyway, if somebody will stumble upon this kind of problem at least there is a workaround.
